src/Codelyzer.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=Codelyzer/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
Any reason this needs to be checked in?

Reply:
This is telling Resharper that the word "Codelyzer" isn't a typo. I got a resharper subscription when I joined and I assumed some other people were using it too. Would it be okay if we configure some solution-specific resharper settings like this in this file? If not, I'm happy to remove this and just keep the settings locally on my machine. 

This setting was just added through the UI popups clicking "add to dictionary -> Solution team-shared"

Reply:
It's a good idea to have this for sure. My only concern here is would this create any issues if someone doesn't have resharper installed?

Reply:
@marknfawaz I’ve merged the latest main branch in with Jon’s fixes and that fixed the failing test. I also ran this on another machine without resharper and the dotsettings file didn’t interfere at all. Let me know if you have any other comments and if you are okay with the comments above that I posted about changing the public surface of the Utils. Jon and I will cut a new release for this once we get this merged into main.